content/tutorials/webinar-revamped-website.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Introducing Cal-Adapt’s New Look and Features
+author: Cal-Adapt
+video: "https://youtu.be/47Mx6aZV1S4"
+tags: data, tools
+category: webinar
+date: "2021-12-09"
+---
+
+Showcases the launch of a revamped Cal-Adapt website and introduces you to its new look and features.
```

Comment:
@elehmer I think this might read a little better if it said "This webinar showcases the launch of..."

Reply:
Agreed. Will edit.